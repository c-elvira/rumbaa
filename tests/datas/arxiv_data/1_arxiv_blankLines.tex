\caption{
	\label{my_label}
	Some comments about table, plot and so on.% test 1
% This line also
	% this line should be removed
	\begin{comment}
		All these line also...
	\end{comment}
}