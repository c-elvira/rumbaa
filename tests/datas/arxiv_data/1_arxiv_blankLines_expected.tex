\caption{
	\label{my_label}
	Some comments about table, plot and so on.
}